app/build.gradle.kts

```
@@ -90,7 +90,7 @@ dependencies {
     implementation("com.squareup.retrofit2:converter-jackson:2.6.4")
     implementation("com.squareup.retrofit2:adapter-rxjava2:2.6.4")
     implementation("com.squareup.retrofit2:converter-scalars:2.1.0")
-    implementation("com.squareup.okhttp3:logging-interceptor:3.12.11")
+    implementation("com.squareup.okhttp3:logging-interceptor:4.9.3")
```

```suggestion
implementation("com.squareup.okhttp3:logging-interceptor:3.12.13")
```
This should not kill retrocompat